config/formats.js

```diff
@@ -23,6 +23,12 @@ let Formats = [
 		team: 'random',
 		ruleset: ['PotD', 'Pokemon', 'Sleep Clause Mod', 'HP Percentage Mod', 'Cancel Mod'],
 	},
+	{
+		name: "[Gen 7] Random Battle (Blitz)",
+		mod: 'gen7',
+		team: 'random',
```

Comment:
`team` should probably also be settable by `ruleset`.

Reply:
I have a WIP branch to make more things ruleset-settable, but OOS for this